frontend/src/Components/Header/Header.js

```diff
@@ -1,37 +1,50 @@
 import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
+import { ReactComponent as ArrowIcon } from '../../assets/icons/arrowicon.svg'
 
-export default function Header({ className, children, secondaryStyle }) {
+export default function Header({
+  className,
+  children,
+  secondaryStyle,
+  hasArrow,
```

Comment:
`hasArrow` is not very precise. What it acually means (especially outside of the component)would be something like "showBackButton" or "isReturnAllowed" etc.
Maybe consider renaming it, to make it more specific.